lib/ansible/plugins/callback/nrdp.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+# (c) 2018 Remi Verchere <remi@verchere.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: nrdp
+    type: notification
+    author: "Remi VERCHERE (@rverchere)"
+    short_description: post task result to a nagios server through nrdp
+    description:
+        - this callback send playbook result to nagios
+        - nagios shall use NRDP to recive passive events
+        - the passive check is sent to a dedicated host/service for ansible
+    version_added: 2.5
```

Comment:
Hello and thank you for your PR! Sorry for the lag time on this but this would need to be updated to `2.6` if you can get it done today, otherwise it will be `2.7` because today is dev freeze for `2.6`.

Reply:
Done!